package.json

```diff
@@ -37,7 +37,7 @@
     "test:e2e:firefox:grid": "mocha --firefox --grid",
     "test:e2e:opera": "mocha --opera",
     "test:e2e:safari": "mocha --safari",
-    "test:server:start": "http-server &",
+    "test:server:start": "http-server -p 8080 &",
```

Comment:
Is dit nodig?

Reply:
Nee niet noodzakelijk, maar als er nu al een http-server werd opgestart met poort 8080, neemt die 8081, en dan zou ik toch liever een foutmelding hebben